src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -27,7 +27,8 @@
 from turicreate.toolkits._internal_utils import (_raise_error_if_not_sframe,
                                                  _numeric_param_check_range)
 
-def create(dataset, target, feature = None, model = 'resnet-50', max_iterations = 10, verbose = True):
+def create(dataset, target, feature = None, model = 'resnet-50',
```

Comment:
Same comment here.

Reply:
This is a great point. I agree and will push a change.

Done.